src/styles/vs2015.css

```diff
@@ -0,0 +1,119 @@
+/*
+ * Visual Studio 2015 dark style
+ * Author: Nicolas LLOBERA <nllobera at gmail.com>
+ */
+
+.hljs {
+  display: block;
+  overflow-x: auto;
+  padding: 0.5em;
+  background: #1E1E1E;
+  color: #DCDCDC;
+}
+
+.hljs-keyword,
+.hljs-literal,
+.hljs-symbol,
+.hljs-name {
+  color: #569CD6;
+}
+.hljs-link {
+  color: #569CD6;
+  text-decoration: underline;
+}
+
+.hljs-built_in,
+.hljs-type,
+.css .hljs-number {
```

Comment:
There shouldn't be any language-specific rules in a style. One of our features is consistency across languages, and we are happy to sacrifice similarities to what the original style (Visual Studio in this case) does. Could you remove all the language specifiers from the file, please.